spec/presenters/vacancy_presenter_spec.rb

```diff
@@ -7,109 +7,6 @@
 
   describe "#job_advert" do
     let(:vacancy) { build_stubbed(:vacancy, job_advert: "<script> call();</script>Sanitized content") }
-
```

Comment:
we either get rid of that code, or instead of testing job_advert, we test any of the other fields that use that code

Reply:
you have to update this test to use job_advert, change it to test any other field like `how_to_apply`